playground/package.json

```diff
@@ -56,6 +57,7 @@
     "antv-site-demo-rc": "0.2.4",
     "csstype": "^3.0.5",
     "lodash": "^4.17.21",
+    "numeral": "^2.0.6",
```

Comment:
Tips: It's a package that is no longer maintained

Reply:
ok here is only used for demo